packages/xsnap/src/xsnap.js

```diff
@@ -470,7 +470,7 @@ export async function xsnap(options) {
         // ensuring that any previous save stream usage has ended. However we
         // must start the flow before receiving the command's response or the
         // xsnap process would block on a full pipe, causing an IPC deadlock.
-        batonKit.promise.then(maybePipe);
+        batonKit.promise.then(maybePipe, noop);
```

Comment:
maybe the same issue here? I definitely want @mhofman to drive this part, I don't understand the whole "baton" thing very well

Reply:
This one is valid, it silences a potential rejection of the baton that would bottom out somewhere else.